configs/11.0/build.mk

```diff
@@ -1 +0,0 @@
-../10.0/build.mk
```

Comment:
I got confused by where the replies are placed, are you talking about my comment regarding scripts/utilities/pkg_build_deb.sh?

If so the problem is that 'echo "${at_dest}/share/modules/modulefiles/${at_ver_rev_internal}"' puts the module file on old AT releases as well, dh_install will not find it (as it doesn't exist on AT < 11.0) thus breaking the build.